signer/migrations/0013__consolidate_withdrawal_tables.sql

```diff
@@ -0,0 +1,10 @@
+
+DROP TABLE sbtc_signer.withdrawal_create_events;
+
+-- Remove the FK `block_hash` REFERENCES `stacks_blocks(block_hash)` constraint.
+ALTER TABLE sbtc_signer.withdrawal_requests
+    DROP CONSTRAINT withdrawal_requests_block_hash_fkey;
+
+-- Add the new column to the `withdrawal_requests` table.
```

Comment:
I would change the comment to be something like the one that you added in the rust field.